src/components/Input.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+const TextInput = styled.input`
+  height: 32px;
+  margin: 0 10px;
+  padding: 0 17px;
+  font-size: ${({ theme }) => theme.font.size.small};
+  max-width: 336px;
```

Comment:
This seems to be specific to the subreddit input. I'd move max-width there

Reply:
oh I missed that good call!